graphdatascience/session/aura_api_responses.py

```diff
@@ -181,6 +180,26 @@ def from_json(cls, json: dict[str, Any]) -> EstimationDetails:
 
         return cls(**{f.name: json[f.name] for f in fields})
 
+    def exceeds_recommended(self) -> bool:
+        return EstimationDetails._parse_size(self.estimated_memory) > EstimationDetails._parse_size(
+            self.recommended_size
+        )
```

Comment:
when would this ever be true?

Reply:
oh, the `recommended_size` is capped by org settings and even our maximum possible size. right, makes sense

Reply:
maximum possible size it is